modules/140-user-authz/monitoring/prometheus-rules/user-authz/deprecated-cluster-authorization-rule-spec-limitnamespaces.tpl

```diff
@@ -0,0 +1,16 @@
+- name: kubernetes.user-authz.deprecated-spec
+  rules:
+  - alert: UserAuthzDeprecatedCARSpecLimitNamespaces
+    expr: >-
+      group by (module) (d8_deprecated_car_spec_limitnamespaces) == 1
```

Comment:
In this case, we usually send a single alert with a command about how to check all deprecated resources.

1. `count(d8_deprecated_car_spec_limitnamespaces) > 0`
2. Add a kubectl command to the description.